src/buildkite_test_collector/collector/constants.py

```diff
@@ -3,4 +3,4 @@
 """This module defines collector-level constants."""
 
 COLLECTOR_NAME='buildkite-test-collector'
-VERSION='1.0.1rc1'
+VERSION='1.0.1'
```

Comment:
Is this right? This is a general release? Have you confirmed it works for the customer?

Reply:
fair point, I changed to rc2